src/appleseed/renderer/modeling/bsdf/oslbsdf.cpp

```diff
@@ -370,6 +377,10 @@ namespace
         auto_release_ptr<BSDF>      m_metal_ggx_brdf;
         auto_release_ptr<BSDF>      m_metal_std_brdf;
         auto_release_ptr<BSDF>      m_orennayar_brdf;
+        auto_release_ptr<BSDF>      m_plastic_beckmann_brdf;
+        auto_release_ptr<BSDF>      m_plastic_ggx_brdf;
+        auto_release_ptr<BSDF>      m_plastic_gtr_brdf;
```

Comment:
`m_plastic_gtr1_brdf`?

Reply:
I think that is the GTR and G1 shadowing/masking from the Disney paper, that @est77 improved.

Reply:
Sure. I'm just wondering if we should add the `1` suffix, seems like @est77 suggested that. Terribly minor detail.